### ISSUE TITLE
Add method to roll ability scores

### ISSUE THREAD

**Issue by msducheminjr** [statelesscode/nerd_dice#7]
As a user I want a convenience method where I can just type `total_ability_scores` and get an array of six ability scores back or `roll_ability_scores` and get an array of `DiceSet` objects back. It should 

```ruby
# return an array of totaled ability scores
NerdDice.total_ability_scores
#=> [12, 14, 13, 15, 10, 8]

# return an array of DiceSet objects including info about the discarded dice
#
NerdDice.roll_ability_scores
#=> [DiceSet0, DiceSet1, ...]
   # => DiceSet0 hash representation { total: 12, dice: [
   #             {value: 2, is_included_in_total: true},
   #             {value: 6, is_included_in_total: true},
   #             {value: 4, is_included_in_total: true},
   #             {value: 1, is_included_in_total: false} 
   #         ]}  
``` 
Default with no arguments will be 4d6 drop the lowest and six ability scores, but allow for options to change the number of sides on the dice, the number of dice rolled, and the number of dice kept.

**Comment by msducheminjr** [comment]
Edited to reflect similar relationship between `total_dice` and `roll_dice`. `total_ability_scores` should return an array of `Integers` and `roll_ability_scores` should return an array of `DiceSets`.

**Comment by msducheminjr** [comment]
Closed by #32
